bridge/src/main/java/com/bitwarden/bridge/util/EncryptionUtils.kt

```diff
@@ -0,0 +1,201 @@
+package com.bitwarden.bridge.util
+
+import com.bitwarden.bridge.IBridgeService
+import com.bitwarden.bridge.IBridgeServiceCallback
+import com.bitwarden.bridge.model.AddTotpLoginItemData
+import com.bitwarden.bridge.model.AddTotpLoginItemDataJson
+import com.bitwarden.bridge.model.EncryptedAddTotpLoginItemData
+import com.bitwarden.bridge.model.EncryptedSharedAccountData
+import com.bitwarden.bridge.model.SharedAccountData
+import com.bitwarden.bridge.model.SharedAccountDataJson
+import com.bitwarden.bridge.model.SymmetricEncryptionKeyData
+import com.bitwarden.bridge.model.toByteArrayContainer
+import kotlinx.serialization.encodeToString
+import java.security.MessageDigest
+import java.security.SecureRandom
+import javax.crypto.Cipher
+import javax.crypto.KeyGenerator
+import javax.crypto.SecretKey
+import javax.crypto.spec.IvParameterSpec
+import javax.crypto.spec.SecretKeySpec
+
+/**
+ * Generate a symmetric [SecretKey] that will used for encrypting IPC traffic.
+ *
+ * This is intended to be used for implementing [IBridgeService.getSymmetricEncryptionKeyData].
+ */
+fun generateSecretKey(): Result<SecretKey> = runCatching {
+    val keygen = KeyGenerator.getInstance("AES")
+    keygen.init(256, SecureRandom())
+    return@runCatching keygen.generateKey()
```

Comment:
Do we need these return calls here? I think we omit these for VaultSdkSourceImpl etc that also use runCatching

Reply:
Yeah good question. 

I added these in 100% for my own readability sake. Looks like other usages of `runCatching` don't use them. Removed:

4a8f906f808c00074bd53db54351aa637a2346b7